flink-cdc-common/src/main/java/com/ververica/cdc/common/utils/InstantiationUtil.java

```diff
@@ -19,7 +19,6 @@
 import org.apache.flink.api.common.typeutils.TypeSerializer;
 import org.apache.flink.core.memory.DataInputViewStreamWrapper;
 import org.apache.flink.core.memory.DataOutputViewStreamWrapper;
-import org.apache.flink.util.CollectionUtil;
```

Comment:
Could you help check other reference usage in our code from flink utils package like `org.apache.flink.util.`? We'd better avoid similar issue.

Reply:
I've made a check and compared the changes from git log with the referenced methods, and found no other incompatible changes.

Reply:
Thanks for the check !